.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+# See https://pre-commit.com for more information
+default_language_version:
+  python: python3.10
+
```

Comment:
Keep scope of python3.10 only on mypy linter (for this case, we could remove this)